build.xml

```diff
@@ -416,48 +412,60 @@
-            </then>
-        </if>
-        <echo message="Put the odbc configuration file in the home folder where CPHP can find it"/>
+    <target name="setup-continuousphp"
+            depends = "check-continuousphp">
+        <echo message="Put the odbc configuration file in the home folder where CPHP can find it" />
         <copy
             file="./vendor/ec-europa/infra/continuousphp/odbc.ini"
             tofile="${user.home}/.odbc.ini"
-            overwrite="true"
-        />
-        <echo message="Allow importing of RDF data from all directories"/>
+            overwrite="true" />
+        <echo message="Allow importing of RDF data from all directories" />
         <exec dir="${project.basedir}"
               command="sudo sed -i 's/DirsAllowed              = ., \/usr\/share\/virtuoso-opensource-6.1\/vad/DirsAllowed = \//g' /etc/virtuoso-opensource-6.1/virtuoso.ini;
 sudo /etc/init.d/virtuoso-opensource-6.1 restart"
               checkreturn="true"
-              passthru="true"/>
-        <echo message="Create Solr index"/>
+              passthru="true" />
+        <echo message="Create Solr index" />
         <exec dir="${project.basedir}"
               command="sudo curl -o /opt/solr/search_api_solr.tar.gz https://ftp.drupal.org/files/projects/search_api_solr-8.x-1.0-alpha4.tar.gz;
 sudo tar xvzf /opt/solr/search_api_solr.tar.gz -C /opt/solr;
 sudo /opt/solr/bin/solr create_core -c drupal;
 sudo mkdir /opt/solr/server/solr/drupal;
 sudo rsync -avz /opt/solr/search_api_solr/solr-conf/5.x/ /opt/solr/server/solr/drupal/conf/"
               checkreturn="true"
-              passthru="true"/>
-        <echo message="Report Solr status"/>
+              passthru="true" />
+        <echo message="Report Solr status" />
         <exec dir="${project.basedir}"
               command="sudo /opt/solr/bin/solr status"
               checkreturn="true"
-              passthru="true"/>
-        <echo message="Restart Solr"/>
+              passthru="true" />
+        <echo message="Restart Solr" />
         <exec dir="${project.basedir}"
               command="sudo /opt/solr/bin/solr restart"
               checkreturn="true"
               passthru="true"
-              spawn="true"/>
+              spawn="true" />
+    </target>
+
+    <target name="check-continuousphp">
+        <echo message="Check if we are running on ContinuousPHP" />
+        <if>
+            <not>
+                <available file="/home/cphp" type="dir" property="environment.cphp" />
+            </not>
+            <then>
+                <fail message="Only run this target on a ContinuousPHP test environment" />
+            </then>
+        </if>
+    </target>
+
+    <target name="build-continuousphp"
+            depends = "check-continuousphp">
+        <echo message="Install the SASS compiler" />
+        <exec dir="${project.basedir}"
+              command="gem install sass"
```

Comment:
We discussed this in person and agreed that pinning is not necessary, as the chances of differences introduced by a new version are really limited.